app/service/data_svc.py

```diff
@@ -168,9 +167,18 @@ def __init__(self):
                         phases.insert(last_phase + 1, [phases[phase_id][idx]])
                         del phases[phase_id][idx + 1:]
             phase_id += 1
-
         return dict(pp)
 
+    async def _load(self):
+        try:
+            for plug in [p for p in await self.locate('plugins') if p.data_dir]:
+                await self._load_abilities(plug)
+                await self._load_adversaries(plug)
+                await self._load_sources(plug)
+                await self._load_planners(plug)
+        except Exception as e:
+            self.log.debug(repr(e), exc_info=True)
```

Comment:
I would change the log line here to error so that the exception doesn't get eaten